lldb/test/API/functionalities/ambigous_commands/TestAmbiguousCommands.py

```diff
@@ -24,7 +24,7 @@ def test_ambiguous_command_with_alias(self):
         self.assertFalse(result.Succeeded())
         self.assertEqual(
             result.GetError(),
-            "Ambiguous command 'co'. Possible matches:\n\tcommand\n\tcontinue\n\tcorefile\n",
+            "error: Ambiguous command 'co'. Possible matches:\n\tcommand\n\tcontinue\n\tcorefile\n",
```

Comment:
This should be "error: ambiguous..." right?

Reply:
Yup, already put up https://github.com/llvm/llvm-project/pull/171519. I'll do the same for the newline.